core/genesis.go

```diff
@@ -250,6 +250,7 @@ func SetupGenesisBlockWithOverride(db ethdb.Database, genesis *Genesis, override
 		if err != nil {
 			return genesis.Config, common.Hash{}, err
 		}
+		performConfigOverride(genesis.Config, overrideGrayGlacier, overrideTerminalTotalDifficulty)
```

Comment:
perhaps we should clone genesis.Config before overriding it?

Reply:
0x62044d2cab405388891ee6d53747817f34c0f00341cde548c0ce9834e9718f27

Reply:
0x62044d2cab405388891ee6d53747817f34c0f00341cde548c0ce9834e9718f27